templates/zerver/help/include/rest-endpoints.md

```diff
@@ -29,6 +29,7 @@
 * [Archive a stream](/api/archive-stream)
 * [Get topics in a stream](/api/get-stream-topics)
 * [Topic muting](/api/mute-topic)
+* [Remove a topic](/api/delete-topic)
```

Comment:
These should always use the same language we use as the `summary` for the page: "Delete a topic".